lib/src/core/atom_derivation.dart

```diff
@@ -100,24 +99,24 @@ class Atom {
 enum DerivationState {
   // before being run or (outside batch and not being observed)
   // at this point derivation is not holding any data about dependency tree
-  NOT_TRACKING,
+  notTracking,
```

Comment:
Is that the convention for enum values in dart? I only picked up dart few weeks back but a long time JavaScript/TypeScript/C#/Java/Swift programmer ;-)

Reply:
Yeah, never seen it in other languages, but for example Flutter code base uses this convention